docs/reference/api/docker_remote_api.md

```diff
@@ -122,6 +122,7 @@ This section lists each version from latest to oldest.  Each listing includes a
 * `GET /events` now supports a `reload` event that is emitted when the daemon configuration is reloaded.
 * `GET /events` now supports filtering by daemon name or ID.
 * `GET /images/json` now supports filters `since` and `before`.
+* `GET /containers/json` now takes network name/s as one of the filters. Contianers connected to the provided network/s will be listed
```

Comment:
I think we can simplify to;

```Markdown
*  `GET /containers/json` now supports filtering containers by `network` name.
```

Don't think we need to go into detail, users should be able to find those in the other documentation